.github/scripts/check_quotes.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# Note: this script was copied from docs-csm:
+# https://github.com/Cray-HPE/docs-csm/blob/05236b40427b46fc837674967891d9d5e754ae73/runLint.sh
+function check_quotes {
+    local error=0
+    printf "=============== Linting \” (https://www.compart.com/en/unicode/U+201C and U+201D) ... \n"
+    grep -n -R \” *.md && echo >&2 'Malformed quotes detected (bad: ” vs. good: ").' && error=1
```

Comment:
I'm just curious, why does this check only look for the right double quotation mark (U+201D) and not the left double quotation mark (U+201C)?

Reply:
I'm not sure. https://github.com/Cray-HPE/docs-csm/commit/05236b40427b46fc837674967891d9d5e754ae73 seems to have changed the print message to say that it was checking for both characters, but the `grep` command didn't change. It looks pretty simple to fix.

Reply:
I'm also noticing now that it uses recursive `grep` but specifies the glob pattern `*.md`, which as a result only scans markdown files in the top-level directory. We ought to fix that, imo.

Reply:
Yeah, makes sense to fix

Reply:
Also submitting https://github.com/Cray-HPE/docs-csm/pull/2533